flang/test/Examples/hello.f90

```diff
@@ -3,7 +3,7 @@
 ! default when they are available.
 
 ! RUN: bbc %s -o - | tco | llc --relocation-model=pic --filetype=obj -o %t.o
-! RUN: %CC %t.o -L%L -lFortranRuntime -lFortranDecimal -lm -o hello
+! RUN: %CC %t.o -L%L -lFortranRuntime -lFortranDecimal -lFortran_main -lm -o hello
 ! RUN: ./hello | FileCheck %s
```

Comment:
I was not sure what expectations were but was aware of it. I did not have the bandwidth to look into the lit configurations.

I see that we already define config.flang_lib_dir and subsitute %L with it. We then use -L%L on the link line. So an easy fix would be to also add -Wl,rpath=%L. This is what I have checked in.

As much as possible, I would like to avoid doing  cut-and-paste of lit config files.

Reply:
Your solution looks good to me.